### DISCUSSION TITLE
Entità timer e calcoli sul tempo residuo

### DISCUSSION THREAD

**Discussion by domoticamoro** [indomus/forum#1876] [HUB: Home Assistant]
Un pò di premessa,scusate la lunghezza....Problema di novembre.
Lavastoviglie non domotica,Bosch.Ha un difetto di fabbrica dove ogni due per tre si brucia la resistenza fusibile di protezione
della scheda.
Ora il costo è irrisorio,identificato il difetto ho estratto la resistenza dalla scheda saldando dei cavi e montandola a morsetti
in una scatola stagna dove è accessibile,quindi se ogni tot mesi ho il guasto la riparo in trenta secondi.
Siccome in zona ci sono spesso sbalzi di tensione tali per cui l’ups sul nas interviene per uno due secondi massimo,
questo un paio di volte la settimana,ho preferito spendere 10€ per una presa domotica in modo da togliere tensione alla lavastoviglie,
che di fatto è l’unica che risente degli sbalzi.

Il timer mi serve per quello,se siamo a casa accendo presa domotica,metto timer a tre ore e quello mi fa spegnere con automazione
la presa.Se sono fuori casa uso il timer a 8 ore in modo che la lavastoviglie ha il suo timer e parte dopo un tot e alla fine del ciclo la presa viene spenta.Fin qui funziona perfettamente per come l’ho creata l’automazione.

**Comment by Scialla** [comment]
Se hai una presa domotica che misura l'assorbimento, la puoi far spegnere quando la lavastoviglie ha finito, senza inventarti cose strane con i timer

**Reply by domoticamoro** [reply]
E' una tapo da 8€ quindi non ha il controllo assorbimenti.
Il sistema di timer e automazione per spegnimento presa funziona perfetto,anche perchè è facilmente impostabile da interfaccia,
ho usato flipdown timer per usarlo a piacimento in base alle ore necessarie e l'automazione ovviamente fà tutto da sè,una volta che il timer ha finito.

Volevo appunto migliorare la gestione attuale avendo degli avvisi per i carichi facendo conti sulla base del timer o del suo tempo residuo.

Creare un sensore in ha che legga il tempo residuo e poi fare calcoli su quello?
Il sensore a dire il vero l'ho anche creato ma come sopra non so come si fanno i calcoli nei template tra un orario e l'altro oppure come si toglie un valore da un campo ora.
